### ISSUE TITLE
Heatmap for informative and non-informative genes

### ISSUE THREAD

**Issue by shahk28** [xuranw/MuSiC#6]
Hello.

I am trying to use MuSIC and want to plot the heatmap of informative and non-informative genes.  Can you guide me on how to extract that matrix?

Thank you,
Kalpit

**Comment by xuranw** [comment]
Hi Kapit,

Thanks for using MuSiC.

In our paper, we selected gene _GC_ and _TTR_ as examples of non-informative and informative genes. 

By the definition of informative and non-informative gene, we can simply look at the `Sigma` matrix returned by function `music_basis()`. The genes with low variance for all cell types (in the columns) are the genes that are informative. However, this matrix does not select genes that have high cross-cell type variance. This means that we probably looking at the genes that are consistently have low expression. Therefore, I suggest to do the ANOVA test and select genes by F-statistics by function `Anova_info()`.

I'm not sure what kind of heatmap you want to plot. The boxplots we used in the supplementary are plotted by function `Relative_gene_boxplot()`.

Hope that helps. 

Best,
Xuran
